src/pybind/mgr/dashboard/frontend/src/app/shared/api/rbd-mirroring.service.ts

```diff
@@ -60,6 +60,32 @@ export class RbdMirroringService {
     return this.http.put(`api/block/mirroring/pool/${poolName}`, request, { observe: 'response' });
   }
 
+  getSiteName() {
+    return this.http.get(`api/block/mirroring/site_name`);
+  }
+
+  setSiteName(siteName) {
+    return this.http.put(
+      `api/block/mirroring/site_name`,
+      { site_name: siteName },
+      { observe: 'response' }
+    );
+  }
+
+  createBootstrapToken(poolName) {
```

Comment:
(not specific only to this method)
I think we should add `@cdEncode` to this class.
Although the UI doesn't allow creating/editing a RBD pool with a `/` in the name, it is still possible to do it via CLI.
If the happens, some of these calls will fail.